src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task{
```

Comment:
Perhaps you can look into using the auto formatting feature in Intellj to standardise the spaces after the curly brackets.